python/py_vapid/__init__.py

```diff
@@ -116,5 +131,5 @@ def sign(self, claims, crypto_key=None):
         else:
             crypto_key = pkey
 
-        return {"Authorization": "Bearer " + sig.strip('='),
+        return {"Authorization": "WebPush " + sig.strip('='),
```

Comment:
Google has changed according to [their docs](https://developers.google.com/web/fundamentals/engage-and-retain/push-notifications/sending-messages#authorization-header)